### PR TITLE
Rules

### PR DESCRIPTION
**Pull request by summonr**
Refactored component layout
Added rules page
responsiveness for rules added

### CODE REVIEW

**Review by endormi — APPROVED** [review]
Well written 👍

**Comment by summonr** [comment]
There is an issue with the routing

File: src/App.js [hunk 1]
```diff
@@ -14,13 +14,13 @@ function App() {
   return (
     <Router>
       <div className="App">
-        <Navbar />
+        {/* <Navbar /> */}
```
Comment (endormi) [inline src/App.js]: Fix <Navbar />

File: src/App.js [hunk 2]
```diff
@@ -14,13 +14,13 @@ function App() {
...
         <Route exact path="/devisle" component={Main} />
         <Route exact path="/rules" component={Rules} />
         <Route exact path="/contributors" component={Contributors} />
         <Route exact path="/schedule" component={Schedule} />
         <Route exact path="/projects" component={Projects} />
-        <Footer />
+        {/* <Footer /> */}
```
Comment (endormi) [inline src/App.js]: Fix <Footer />

Comment (summonr) [reply src/App.js]: removed

**Comment by summonr** [comment]
the links directly do not work, 404 if rules are accessed for example: https://devisle.github.io/rules

File: src/App.js (hunk 1 above)
Comment (summonr) [reply src/App.js]: fixed

**Comment by endormi** [comment]
I tested the branch, I ran the server and for some reason it didn't show me any content.

**Pull request merged**